src/tezos_interop/tezos_js_bridge.js

```diff
@@ -193,6 +201,41 @@ const onStorageRequest = async (id, content) => {
   respond(id, { status: "success", storage });
 };
 
+/** @param {BigMapRequest} content */
+const onBigMapRequest = async (id, content) => {
+  const { rpc_node, confirmation, destination, key } = content;
+  const client = new RpcClient(rpc_node);
+  const Tezos = new TezosToolkit(rpc_node);
+  Tezos.setProvider({ config });
+
+  const block = await client.getBlock(); // fetches the head
+
+  const value = await client.getBigMapKey(
```

Comment:
https://github.com/ecadlabs/taquito/blob/9043379b3d87310892fc9d8ab7c9c14cd8b7a9f4/docs/version.md

This function is deprecated